src/odemis/gui/cont/acquisition/fastem_acq.py

```diff
@@ -283,15 +425,21 @@ def on_acquisition_done(self, future, num):
 
         # Store DataArray as TIFF in pyramidal format and reopen as static stream (to be memory-efficient)
         # TODO: pick a different name from previous acquisition?
```

Comment:
I think this TODO can be removed, it's fine that the overview image gets overwritten, especially now that it is saved in a directory per user.